build.gradle

```diff
@@ -535,6 +541,24 @@ configure(subprojects - getJsProjects()) { subproject ->
     }
 }
 
+def checkDexer = task('checkDexer', description: 'compiles android class files into dex bytecode', dependsOn: compileKotlin) {
+    getAndroidProjects().each { p->
+        doLast {
+            javaexec {
+                def nameWithoutSuffix = p.name.substring(p.name.indexOf('-') + 1, p.name.lastIndexOf('-'))
+                def jvmProject = prefixedProject(nameWithoutSuffix + '-jvm')
+                def jarToTest = "${jvmProject.projectDir}\\build\\libs\\*.jar"
```

Comment:
what is the reason that you test the jvm jar? We don't guarantee that the jvm jar is dex compatible but  the android jar should be.

Reply:
Also, please use forward slashes

Reply:
I noticed this method on the build.gradle of android projects so I figured we use jvm projects jar
But now I saw that building with android switch create jars so we can use them

Problem is that the build fails for me